lib/resources/security_policy.rb

```diff
@@ -138,7 +138,7 @@ def read_params
 
       conf = SimpleConfig.new(
         @content,
-        assignment_regex: /^\s*(.*)=\s*(\S*)\s*$/,
+        assignment_regex: /^\s*(.*)=\s*([\S[:space:]]*)\s+$/,
```

Comment:
Both the original regexp and the new one are wrong for the given change. If I just critique your change, then `[\S[:space:]]*` is essentially `.*` as `\S` is non-whitespace and `[:space:]` is roughly the same as `\s` which is whitespace. I'd rather it be `.` but also with `+` since a zero length match should be meaningless.

[lo siento... mí Español es malo. Es google:]

Tanto la regexp original como la nueva están equivocadas para el cambio dado. Si solo critico su cambio, entonces `[\S[: space:]]*` es esencialmente `.*` Ya que `\S` es "non-whitespace" y` [:space:] `es aproximadamente lo mismo que`\s` que es espacio en blanco. Prefiero que sea `.` pero también con `+` ya que una coincidencia de longitud cero no debería tener sentido.

Reply:
Tambien, Necesita una prueba.